cli/docker/environment_source.go

```diff
@@ -8,7 +8,7 @@ import (
 )
 
 // implementing env.EnvironmentSource
-type DockerEnvironmentSource struct {
+type EnvironmentSource struct {
```

Comment:
Package name is `docker`, this struct name should not stutter (`docker.DockerSomething`)